contracts/state/State.sol

```diff
@@ -51,6 +51,8 @@ contract State is Ownable2StepUpgradeable, IState {
      */
     bool internal _defaultIdTypeInitialized;
 
+    mapping(bytes2 => bool) internal _typeIdMap;
```

Comment:
I think we should put this mapping into `StateLib.Data` (which supports safe upgrade with __gap) and put __pap to 47 there

Reply:
Also please increase State `VERSION` to `2.5.0`

Reply:
And also bump contracts package.json version to `2.2.0`